### ISSUE TITLE
how can i change my whatsapp privacy setting using wa-js

### ISSUE THREAD

**Issue by silemftooh** [wppconnect-team/wa-js#1934]
how can i get my whatsapp privacy setting
I want to know the settings of the connected WhatsApp account
Is there some way?

**Comment by icleitoncosta** [comment]
There is no option available, but it will be added soon.

**Comment by opsJson** [comment]
You can get and set like this:

```js
const UserPrefsGeneral = require("WAWebUserPrefsGeneral");
const settings = UserPrefsGeneral.getUserPrivacySettings();
settings.lastSeen = "none";
UserPrefsGeneral.setUserPrivacySettings(settings);
```

**Comment by silemftooh** [comment]
@opsJson  
I got it another way, but your solution is much better
Thank you

**Comment by icleitoncosta** [comment]
Functions implemented in v3.8.0
